services/logs/logs.go

```diff
@@ -74,6 +89,37 @@ type Logs struct {
 	ctx            context.Context
 }
 
+type manageConfig struct {
+	Users []struct {
+		Username string `yaml:"username"`
+		Password string `yaml:"password"`
+	} `yaml:"users"`
+}
+
+// Fetch PMM credential
+func getCredential(ctx context.Context) string {
+	u := ""
+	f, err := os.Open("/srv/update/pmm-manage.yml")
+	if err != nil {
+		return u
```

Comment:
It might make sense to handle (log) or return error – the absence of `pmm-manage.yml` is something code should not handle silently.